autogen/main/cluster.tf.tmpl

```diff
@@ -674,15 +673,17 @@ resource "google_container_cluster" "primary" {
       }
       {% endif %}
       {% if autopilot_cluster != true %}
+      {% if beta_cluster %}
       gcfs_config {
         enabled = var.enable_gcfs
       }
       {% endif %}
+      insecure_kubelet_readonly_port_enabled = var.insecure_kubelet_readonly_port_enabled != "" ? var.insecure_kubelet_readonly_port_enabled : null
```

Comment:
Looks like `insecure_kubelet_readonly_port_enabled` is a bool, not string.  Should this be `!= null`?